File: 1421-FindNumbersWithEvenNumberOfDigits/1421-FindNumbersWithEvenNumberOfDigits.py
# Last updated: 6/17/2025, 8:58:06 AM
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        check = []
        for i in nums:
            if len(str(i)) % 2 == 0:
                check.append(len(str(i)))
        return len(check)

        

File: 1421-FindNumbersWithEvenNumberOfDigits/1421-FindNumbersWithEvenNumberOfDigits.java
// Last updated: 7/19/2025, 7:30:35 PM
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int i=0; i<nums.length; i++){
            if(IsEven(nums[i])){
                count++;
            }
        }
        return count;
    }
    public boolean IsEven(int i){
        int count = 0;
        while(i>0){
            i=i/10;
            count++;
        }

        if(count % 2 == 0){
             return true;
        }
    return false;
    }
}